releases/prometheus-operator.yaml

```diff
@@ -296,6 +296,17 @@ releases:
                     firing: '{{"{{"}} template "pagerduty.default.firing" . {{"}}"}}'
                     resolved: '{{"{{"}} template "pagerduty.default.resolved" . {{"}}"}}'
               {{- end }}
+              {{- if not ( env "KUBE_PROMETHEUS_ALERT_MANAGER_OPSGENIE_API_KEY" | empty ) }}
+              opsgenie_configs:
+                - api_key: '{{ env "KUBE_PROMETHEUS_ALERT_MANAGER_OPSGENIE_API_KEY" | empty }}'
+                  send_resolved: true
+                  {{- if not ( env "NAMESPACE" | empty ) }}
+                  tags: '{{ env "NAMESPACE" | default "" }}'
```

Comment:
can we pass the FQHN of the cluster?

Reply:
We have many variables, with FQDN that you are looking for we have this:
DOCKER_NAME = staging.cathex.io
KOPS_CLUSTER_NAME = eu-west-2.staging.cathex.io
I could use them but I’m not sure if that’s the best option as one is KOPS and it’s not the only way we spin up K8S clusters. I can instead provide not required variable so people can set FQDN themselves.